src/System.Net.Sockets/src/System/Net/Sockets/UdpReceiveResult.cs

```diff
@@ -4,12 +4,19 @@
 
 namespace System.Net.Sockets
 {
-    public struct UdpReceiveResult : IEquatable<UdpReceiveResult>
-    {
+	/// <summary>
```

Comment:
CoreFX [coding style](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md) is to use four spaces, not tabs.

Reply:
Auto changed by VSSettings. Changed